picard/coverart/processing/__init__.py

```diff
@@ -41,5 +56,19 @@ def run_image_metadata_filters(metadata):
     return True
 
 
-# def run_image_processors(data):
-#     pass
+def run_image_processors(data, coverartimage):
+    try:
+        info = ImageInfo(*imageinfo.identify(data))
```

Comment:
Instead `imageinfo.identify()` could directly return ImageInfo tuple and move the ImageInfo declaration there.

https://github.com/metabrainz/picard/blob/cc3801059734b4d6cca0a3fd51b07666d0c5309f/picard/util/imageinfo.py#L63